SwiftSampleProject/SwiftyVokoder/Models/Station.swift

```diff
@@ -56,17 +56,17 @@ extension Station: VOKMappableModel {
         return { (_, inputObject: NSManagedObject) in
             guard let
```

Comment:
With Swift 3 syntax I wonder if we should be bringing the `let` down now for better alignment?